cmd/thanos/config.go

```diff
@@ -189,6 +190,8 @@ func (qc *queryConfig) registerFlag(cmd extkingpin.FlagClause) *queryConfig {
 		Default("POST").EnumVar(&qc.httpMethod, "GET", "POST")
 	cmd.Flag("query.sd-dns-resolver", "Resolver to use. Possible options: [golang, miekgdns]").
 		Default("golang").Hidden().StringVar(&qc.dnsSDResolver)
+	cmd.Flag("query.default-step", "Default range query step to use. This is only used in stateless Ruler and alert state restoration.").
```

Comment:
Why is this so small? :thinking:

Reply:
I am not sure whether we want to expose this to users or not. This can be calculated simply by `max(for-outage-tolerance / 250, 1s)`. But in E2E test we might still want to modify it.

For the default value `1s`, I just use the same default query step set in `query.go`.